elementary/config/config.py

```diff
@@ -13,18 +13,20 @@ class Config:
 
     DEFAULT_CONFIG_DIR = str(Path.home() / '.edr')
     DEFAULT_PROFILES_DIR = str(Path.home() / '.dbt')
+    DEFAULT_TARGET_DIR = 'output'
 
     def __init__(self, config_dir: str = DEFAULT_CONFIG_DIR, profiles_dir: str = DEFAULT_PROFILES_DIR,
                  profile_target: str = None, update_bucket_website: bool = None, slack_webhook: str = None,
                  slack_token: str = None, slack_channel_name: str = None, aws_profile_name: str = None,
                  aws_access_key_id: str = None, aws_secret_access_key: str = None, s3_bucket_name: str = None,
-                 google_service_account_path: str = None, gcs_bucket_name: str = None):
+                 google_service_account_path: str = None, gcs_bucket_name: str = None,
+                 target_dir: str = DEFAULT_TARGET_DIR):
         self.config_dir = config_dir
         self.profiles_dir = profiles_dir
         self.profile_target = profile_target
         config = self._load_configuration()
 
-        self.target_dir = config.get('target-path') or os.getcwd()
+        self.target_dir = config.get('target-path') or target_dir
```

Comment:
I tend to agree that it would be a good idea to put our outputs under a single directory by default rather than the working directory, but we'll need to discuss this change as users might currently rely on the path of the report and such.

Reply:
Correct, lmk what you think, in general I assume the flat working dir might giving the user headache anyway as they might move and cleanup.. we can also use the os.getcwd() as the default

Reply:
Agree with both of you :) maybe we can start with default as os.getcwd(). Then we can announce in the community that we are going to change the default from cwd to 'edr_output' and deploy this change afterwards?

Reply:
Yup, good point